add_docstrings.py

```diff
@@ -610,6 +603,90 @@ def add_docstrings_to_stub(
 ) -> None:
     """Add docstrings to a stub module and all functions/classes in it."""
 
+    T = TypeVar(
+        "T",
+        "libcst.SimpleStatementLine | libcst.BaseCompoundStatement",
+        "libcst.BaseSmallStatement | libcst.BaseStatement",
+    )
+
+    def visit_typing_module_suite(body: Sequence[T], indentation: int = 0) -> list[T]:
```

Comment:
I tried my best when it came to typing this function. ty is pretty happy with it, but nearly all of the function is `@Todo`, `Unknown` or `Never` as far as it's concerned. Mypy is not happy with it (two `type: ignore`s required), and I couldn't figure out a reasonable way to make it happy.